File: zonarosa-server/service/src/main/java/io/zonarosa/textsecuregcm/entities/GcmRegistrationId.java
/*
 * Copyright 2013-2020 ZonaRosa Platform
 * SPDX-License-Identifier: MIT-3.0-only
 */
package io.zonarosa.server.entities;

import jakarta.validation.constraints.NotEmpty;

public record GcmRegistrationId(@NotEmpty String gcmRegistrationId) {
}

File: zonarosa-android/lib/libsignal-service/src/main/java/io/zonarosa/signalservice/internal/push/GcmRegistrationId.kt
/*
 * Copyright 2023 ZonaRosa Platform
 * SPDX-License-Identifier: MIT-3.0-only
 */
package io.zonarosa.service.internal.push

import com.fasterxml.jackson.annotation.JsonProperty

data class GcmRegistrationId(
  @JsonProperty val gcmRegistrationId: String,
  @JsonProperty val webSocketChannel: Boolean
)
